lgc/include/lgc/state/AbiMetadata.h

```diff
@@ -550,7 +550,6 @@ static constexpr char PosFixedPtEna[] = ".pos_fixed_pt_ena";
 // User data mapping for special user data values.
 enum class UserDataMapping : unsigned {
   GlobalTable = 0x10000000,          // 32-bit pointer to GPU memory containing the global internal table.
-  PerShaderTable = 0x10000001,       // 32-bit pointer to GPU memory containing the per-shader internal table.
```

Comment:
We should try to keep this aligned with what's in PAL. Is this actually being removed entirely from PAL?

Reply:
As far as I know, SC still relies on it.

Reply:
I am happy with removing it completely from LLPC. If Nicolai prefers, perhaps keep the enum here, but remove everything else.

Reply:
Yeah, let's keep the enum but remove everything else.

Reply:
Thank you, all. I will follow your advice.